pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/samples/crd_sample.go

```diff
@@ -53,6 +53,8 @@ func (f *CRDSample) SetTemplateDefaults() error {
 const crdSampleTemplate = `apiVersion: {{ .Resource.QualifiedGroup }}/{{ .Resource.Version }}
 kind: {{ .Resource.Kind }}
 metadata:
+  labels:
+    app.kubernetes.io/name: {{ lower .Resource.Kind }}-sample
```

Comment:
Could we not add the rest of labels here? 

See the example: 

```
   app.kubernetes.io/name: mysql  -> this is the Kind 
    app.kubernetes.io/instance: mysql-abcxzy -> this is the name of the instance 
    app.kubernetes.io/part-of: Wordpress --> project
    app.kubernetes.io/created-by: controller-manager
```

Reply:
updated in `kustomize/v1` plugin in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/8a2c1d7c48ec7743828b14a04ae9de765cde30b2

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.